integration/spark/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchemaCommon.scala

```diff
@@ -624,9 +625,7 @@ class TableNewProcessor(cm: TableModel) {
     if (isVarcharColumn(colName)) {
       columnSchema.setDataType(DataTypes.VARCHAR)
     }
-    // TODO: Need to fill RowGroupID, converted type
```

Comment:
is this `TODO` is completed/not required?

Reply:
not required now